doc/userguide/output/eve/eve-json-format.rst

```diff
@@ -147,13 +141,76 @@ the signature.
     }
   },
 
+Action field
+~~~~~~~~~~~~
+
+Possible values: "allowed" and "blocked".
+
+Example:
+
+::
+
+  "action":"allowed"
+
+Action is set to "allowed" unless a rule used the "drop" action and Suricata is
+in IPS mode, or when the rule used the "reject" action. It is important to note
+that this does not necessarily indicate the final verdict for a given packet or
+flow, since one packet may match on several rules.
+
+Verdict Field
```

Comment:
this is not a field, but an object containing other fields, right?

Reply:
Right, I was seeing it as a field because it's part of 'alert', but... that's indeed wrong.

Reply:
also: update description, there are more fields in verdict now.